lnrpc/rpc.proto

```diff
@@ -94,13 +94,15 @@ service WalletUnlocker {
 message GenSeedRequest {
     /**
     aezeed_passphrase is an optional user provided passphrase that will be used
-    to encrypt the generated aezeed cipher seed.
+    to encrypt the generated aezeed cipher seed.  When using REST, this field
```

Comment:
Nit: there's a double space after the period here and in some other cases below that would be reflected in the API documentation.

Reply:
The double space was used in several places so I thought that was a style we wanted to use for some reason. But I removed all of them now.